### ISSUE TITLE
db_subnet_group_name must be specified when creating aws_rds_cluster_instance resource

### ISSUE THREAD

**Issue by mschurenko** [hashicorp/terraform#3607]
I don't see db_subnet_group_name as a parameter in the documentation:

https://www.terraform.io/docs/providers/aws/r/rds_cluster_instance.html

When my resource was defined like this:

```
resource "aws_rds_cluster_instance" "aurora_cluster_instance" {
  count = 1
  identifier = "matt-aurora-test"
  cluster_identifier = "${aws_rds_cluster.rds_cluster.id}"
  instance_class = "db.r3.large"
}
```

After running "terraform apply" I got this error:

```shell
Error applying plan:

1 error(s) occurred:

* aws_rds_cluster_instance.aurora_cluster_instance: InvalidVPCNetworkStateFault: Cannot create the DB Instance because db subnet group has not been specified which is required for a private DBInstance creation.
	status code: 400, request id: 9bbaa9d3-790f-11e5-88ff-e74326af9414
```

After updating the resource to:
```
resource "aws_rds_cluster_instance" "aurora_cluster_instance" {
  count = 1
  identifier = "matt-aurora-test"
  cluster_identifier = "${aws_rds_cluster.rds_cluster.id}"
  instance_class = "db.r3.large"
  db_subnet_group_name = "${aws_db_subnet_group.db_subnet_group.id}"
}
```

It was created successfully.

Thanks

**Comment by stack72** [comment]
@apparentlymart this is fixed in #3789

**Comment by catsby** [comment]
Thanks for writing in @mschurenko , sorry for the trouble there. @stack72 has updated the docs and I've merged the change.
